src/patternfly/components/Touchspin/touchspin.scss

```
@@ -0,0 +1,55 @@
+.pf-c-touchspin {
+  // main
+  --pf-c-touchspin__main--m-active--c-form-control--Color: var(--pf-global--Color--100);
+
+  // units
+  --pf-c-touchspin__main__units--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-touchspin__units__main--MarginLeft: var(--pf-global--spacer--sm);
+
+  // control
+  --pf-c-touchspin__control--m-threshold--c-button--Color: var(--pf-global--disabled-color--100);
+
+  // icon
+  --pf-c-touchspin__control-icon--FontSize: var(--pf-global--FontSize--xs);
+
+  // form control
+  --pf-c-touchspin--c-form-control--Color: var(--pf-global--disabled-color--100);
+  --pf-c-touchspin--c-form-control--MaxWidth: 6ch;
+
+  display: flex;
```

should this be inline?

```suggestion
  display: inline-flex;
```